Libraries/LibWeb/CSS/Interpolation.cpp

```diff
@@ -611,6 +611,11 @@ NonnullRefPtr<CSSStyleValue const> interpolate_value(DOM::Element& element, Calc
         auto from_ratio = from.as_ratio().ratio();
         auto to_ratio = to.as_ratio().ratio();
 
+        // https://drafts.csswg.org/css-sizing-4/#aspect-ratio
+        // If the <ratio> is degenerate, the property instead behaves as auto.
```

Comment:
No strong feelings on this, but the [interpolation rules]() for `<ratio>`, where the comment below comes from, says at the end:

> If either [<ratio>](https://drafts.csswg.org/css-values-4/#ratio-value) is [degenerate](https://drafts.csswg.org/css-values-4/#degenerate-ratio), the values cannot be interpolated.

Which agrees with your solution, but maybe is better to link to as it applies to all `<ratio>`s instead of just `aspect-ratio`.